docs/identity/enterprise-apps/myapps-overview.md

```diff
@@ -148,6 +148,10 @@ Applications can be hidden from the My Apps portal by a user or administrator. A
 
 For more information, see [Create collections on the My Apps portal](access-panel-collections.md).
 
+## Domain Hint
+
+In case there is domain federation and in order for user to be redirected to external federation endpoint for authentication, request to My Apps must contain a domain hint URL parameter such as "domain_hint=example.com".
```

Comment:
```suggestion
> [!IMPORTANT]
>In case there is domain federation and for the user to be redirected to an external federation endpoint for authentication, the request to My Apps must contain a domain hint URL parameter such as "domain_hint=example.com".
```

Reply:
The domain hint section doesn't seem worthy of a level 2 heading because it's very small. It's also not one of the main points of focus for this article. Publishing it as is won't be good for SEO because domain hint isn't one of the key terms we want this article to be discoverable for. So, I suggest using an alert box instead as shown in my suggestion.